src/main/java/de/zalando/aruha/nakadi/controller/EventStreamController.java

```diff
@@ -71,7 +75,13 @@ public StreamingResponseBody streamEvents(
             final NativeWebRequest request, final HttpServletResponse response) throws IOException {
 
         return outputStream -> {
+
+            Counter consumerCounter = null;
+
             try {
+                consumerCounter = metricRegistry.counter(metricNameFor(eventTypeName, CONSUMERS_COUNT_METRIC_NAME));
+                consumerCounter.inc();
```

Comment:
I'm not sure if it should be increased here, because at this point the stream is not really started yet. There are still some checks to be performed. But that is of course a point to argue what is considered to be a started stream.